curvefs/src/mds/metaserverclient/metaserver_client.cpp

```diff
@@ -69,11 +72,18 @@ FSStatusCode MetaserverClient::CreateRootInode(uint32_t fsId, uint32_t poolId,
     request.set_mode(mode);
 
     stub.CreateRootInode(&cntl, &request, &response, nullptr);
-
+    uint32_t maxRetry = options_.rpcRetryTimes;
+    while (cntl.Failed() && maxRetry > 0) {
```

Comment:
missing check response is ok, and another problem is leader may changed, you should deal with this case.